app/src/main/java/ch/sdp/vibester/user/User.kt

```diff
@@ -9,5 +9,6 @@ data class User(
     var totalGames: Int = 0,
     var bestScore: Int = 0,
     var correctSongs: Int = 0,
-    var ranking: Int = 0
-) : Serializable {}
\ No newline at end of file
+    var ranking: Int = 0,
+    var uid: String = ""
```

Comment:
why do we need an uid if the handle is already unique for each user?

Reply:
I was not sure whether the handle is correct. I saved the uid since the user is stored by it. We can discuss it next meeting.